scripts/build-mac.sh

```diff
@@ -12,9 +12,19 @@ cd $dirname/..
 PYTHONPATH=
 pip3 install -r requirements.txt
 python3 setup.py bdist_mac
-cd build/*.app/Contents
-cp -r MacOS/ ../../pgtoolsservice
+
+# Compress mysqltoolsservice folder
+cd $pwd/build
+cp mysqltoolsservice/lib* mysqltoolsservice/lib/.
```

Comment:
Add a comment here.

TODO : Remove this fix done for openssl libraries need to be copied manually in lib folder in mac